samples/hello/src/com/microsoft/aad/adal/hello/Constants.java

```diff
@@ -28,20 +28,20 @@
 
     public static final String HEADER_AUTHORIZATION = "Authorization";
 
-    public static final String HEADER_AUTHORIZATION_VALUE_PREFIX = "Bearer ";
+    public static final String HEADER_AUTHORIZATION_VALUE_PREFIX = "Bearer";
     
     // AAD PARAMETERS
     // https://login.windows.net/tenantInfo
-    static final String AUTHORITY_URL = "https://login.windows.net/common";
+    static final String AUTHORITY_URL = "https://login.microsoftonline.com/msdevex.onmicrosoft.com";
```

Comment:
Do you mean we need to keep the variables in constants.java blank/to-be-defined?

From: Wei Jia [mailto:notifications@github.com]
Sent: Tuesday, December 8, 2015 11:06 AM
To: AzureAD/azure-activedirectory-library-for-android <azure-activedirectory-library-for-android@noreply.github.com>
Cc: Heidi Jinxu Jia <jinjia@microsoft.com>
Subject: Re: [azure-activedirectory-library-for-android] Debugged two lint error (#459)


In samples/hello/src/com/microsoft/aad/adal/hello/Constants.java<https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/459#discussion_r46997796>:

>

>      // AAD PARAMETERS

>      // https://login.windows.net/tenantInfo

> -    static final String AUTHORITY_URL = "https://login.windows.net/common";

> +    static final String AUTHORITY_URL = "https://login.microsoftonline.com/msdevex.onmicrosoft.com";

Can we remove those testing data info before merging to dev?

—
Reply to this email directly or view it on GitHub<https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/459/files#r46997796>.